src/funcCoreTools/getLocalFuncCoreToolsVersion.ts

```diff
@@ -26,3 +27,15 @@ export async function getLocalFuncCoreToolsVersion(): Promise<string | null> {
         return null;
     }
 }
+
+export function addLocalFuncTelemetry(actionContext: IActionContext): void {
+    actionContext.properties.funcCliVersion = 'unknown';
+
+    // tslint:disable-next-line:no-floating-promises
+    getLocalFuncCoreToolsVersion().then((version: string) => {
+        // tslint:disable-next-line:strict-boolean-expressions
+        actionContext.properties.funcCliVersion = version || 'none';
```

Comment:
There's no guarantee that the property will get set in time to be included in telemetry...

Reply:
I don't think it'll happen very often, but that's why I set `funcCliVersion` to 'unknown' a few lines up (so that I could at least differentiate that case)